src/Microsoft.Diagnostics.Monitoring/Microsoft.Diagnostics.Monitoring.csproj

```diff
@@ -9,7 +9,7 @@
     <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
     <IncludeSymbols>true</IncludeSymbols>
-    <IsShippingAssembly>true</IsShippingAssembly>
+    <IsShippingAssembly>false</IsShippingAssembly>
```

Comment:
`IsPackable` is true. This will make the CI build a nuget package for this. The right attribute here is `IsShipping` which disables promotion of both the assembly and the package. Also, are this and `Microsoft.Diagnostics.Monitoring.LogAnalytics` expected to be nuget packages, or are these only to be used in `dotnet-monitor`?

Reply:
Short term, the assembly will be only consumed by dotnet-monitor.

Reply:
Then can you please set `IsPackable` to false in both the non-shipping products so we don't end up shipping more packages for now?